src/components/Eidas.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import NotificationModal from "../login/components/Modals/NotificationModal";
+import { FormattedMessage } from "react-intl";
+
+interface EidasProps {
+  showModal: boolean;
+  eidas_sp_freja_idp_url: string;
+  handleShowModal: React.MouseEventHandler<HTMLButtonElement>;
+  handleHideModal: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+function Eidas(props: EidasProps) {
+  // Temporary instructions until Sweden Connect has more alternatives and we have a DS
+  const freja_instructions = (
+    <div id="freja-instructions">
+      <ol>
+        <li>{<FormattedMessage description="eidas freja instructions step1" defaultMessage={`Install the app`} />}</li>
+        <li>
+          {
+            <FormattedMessage
+              description="eidas freja instructions step2"
+              defaultMessage={`Create a Freja eID Plus account (awarded the "Svensk e-legitimation" quality mark)`}
+            />
+          }
+        </li>
+        <li>
+          {
+            <FormattedMessage
+              description="eidas freja instructions step3"
+              defaultMessage={`The app will generate a QR-code`}
+            />
+          }
+        </li>
+        <li>
+          {
+            <FormattedMessage
+              description="eidas freja instructions step4"
+              defaultMessage={`Find a local authorised agent, show them a valid ID together with the QR-code and 
+              they will be able to verify your identity`}
+            />
+          }
+        </li>
+        <label>
+          {
+            <FormattedMessage
+              description="eidas freja instruction tip1"
+              defaultMessage={`Tip: Use the app to find your nearest agent`}
+            />
+          }
+        </label>
+        <li>
+          {
+            <FormattedMessage
+              description="eidas freja instructions step5"
+              defaultMessage={`Freja eID is now ready to be used with your eduID`}
+            />
+          }
+        </li>
+      </ol>
+      <a href="https://frejaeid.com/skaffa-freja-eid/" target="_blank">
+        {<FormattedMessage description="eidas freja instructions install link" defaultMessage={`What is Freja eID?`} />}
+      </a>
+    </div>
+  );
+
+  function useFrejaeID(event: React.MouseEvent<HTMLElement>) {
+    event.preventDefault();
+    window.location.href = props.eidas_sp_freja_idp_url;
+  }
+
+  return (
+    <div>
+      <div className="vetting-button">
+        <button id="eidas-show-modal" onClick={props.handleShowModal}>
+          <div className="text">
+            {
+              <FormattedMessage
+                description="verify identity vetting freja tagline"
+                defaultMessage={`For you able to create a Freja eID by visiting one of the authorised agents`}
```

Comment:
I think it should say "Freja eID+" here. Freja eID is another concept which we won't accept currently.